templates/web/base/reports/_list-filters.html

```diff
@@ -34,18 +35,20 @@
 [% END %]
 
         <p class="report-list-filters">
-          [% tprintf(loc('<label for="statuses">Show</label> %s reports <label for="filter_categories">about</label> %s', "The first %s is a dropdown of all/fixed/etc, the second is a dropdown of categories"), mark_safe(select_status), mark_safe(select_category)) %]
+          [% tprintf(loc('<label for="statuses">Show reports</label> %s <label for="filter_categories">About</label> %s', "The first %s is a dropdown of all/fixed/etc, the second is a dropdown of categories"), mark_safe(select_status), mark_safe(select_category)) %]
```

Comment:
We'll have to split this up more than this if the labels are now not within a sentence, no need for it to be solely one translation string any longer. But that will break the translations, but I can't see a way to do it otherwise.

Reply:
Sorry, not sure what you want me to do here.

Reply:
As it's no longer a sentence, each bit can be translated separately, more like standard form labels e.g.
```
<label for="statuses">[% loc('Show reports') %]</label>
[% mark_safe(select_status) %]
<label for="filter_categories">[% loc('About') %]</label>
[% mark_safe(select_category) %]
```